layouts/shortcodes/dbm-sqlserver-agent-config-examples.md

```diff
@@ -93,19 +106,19 @@ instances:
     custom_queries:
     - query: SELECT age, salary, hours_worked, name FROM hr.employees;
       columns:
-        - name: age
+        - name: sqlserver.custom.employee_age
           type: gauge
-        - name: salary
+        - name: sqlserver.custom.employee_salary
            type: gauge
-        - name: hours
+        - name: sqlserver.custom.employee_hours
            type: count
         - name: name
            type: tag
       tags:
         - 'table:employees'
 ```
 ### Working with hosts through a remote proxy
-When connecting to a database host through a remote proxy, it can be useful to set a custom hostname. Utilize the `reported_hostname` option to override the hostname detected by the agent.
+If the Agent must connect to a database host through a remote proxy, all telemetry will be tagged with the hostname of the proxy rather than the database instance. Utilize the `reported_hostname` option to set a custom override of the hostname detected by the agent.
```

Comment:
```suggestion
If the Agent must connect to a database host through a remote proxy, all telemetry is tagged with the hostname of the proxy rather than the database instance. Utilize the `reported_hostname` option to set a custom override of the hostname detected by the Agent.
```